.github/maintainers_guide.md

```diff
@@ -126,65 +144,74 @@ username: {your username}
+      - `git push future-release`
+    - Create a git tag for the release. For example `git tag v1.3.0.dev0`.
+    - Push the tag up to github with `git push origin --tags`
 
-#### Production Deployment
+2. Distribute the release
+   - Use the latest stable Python runtime
+   - `python -m venv env`
+   - `./scripts/deploy_to_pypi_org.sh`
+   - You do not need to create a GitHub release
+
+3. (Slack Internal) Communicate the release internally
 
-1.  Create the commit for the release:
 
-- Bump the version number in adherence to [Semantic Versioning](http://semver.org/) in `slack_bolt/version.py`
-  - `echo '__version__ = "1.2.3"' > slack_bolt/version.py`
-- Commit with a message including the new version number. For example `1.2.3` & Push the commit to a branch and create a PR to sanity check.
-  - `git checkout -b v1.2.3-release`
-  - `git commit -m'version 1.2.3'`
-  - `git push {your-fork} v1.2.3-release`
-- Merge in release PR after getting an approval from at least one maintainer.
-- Create a git tag for the release. For example `git tag v1.2.3`.
-- Push the tag up to github with `git push origin --tags`
+#### Production Deployment
 
-2.  Distribute the release
+1. Create the commit for the release: 
+   - Bump the version number in adherence to [Semantic Versioning](http://semver.org/) in `slack_bolt/version.py`
+   - Commit with a message including the new version number. For example `1.2.3` & Push the commit to a branch and create a PR to sanity check.
+     - `git checkout -b v1.2.3-release`
+     - `git commit -m'version 1.2.3'`
+     - `git push {your-fork} v1.2.3-release`
+   - Merge in release PR after getting an approval from at least one maintainer.
+   - Create a git tag for the release. For example `git tag v1.2.3`.
+   - Push the tag up to github with `git push origin --tags`
 
-- Use the latest stable Python runtime
-- `python -m venv env`
-- `./scripts/deploy_to_prod_pypi_org.sh`
-- Create a GitHub release - https://github.com/slackapi/bolt-python/releases
+2. Distribute the release
+   - Use the latest stable Python runtime
+   - `python -m venv env`
+   - `./scripts/deploy_to_pypi_org.sh`
+   - Create a GitHub release - https://github.com/slackapi bolt-python/releases
 
-```markdown
-## New Features
+   ```markdown
+   ## New Features
 
-### Awesome Feature 1
+   ### Awesome Feature 1
 
-Description here.
+   Description here.
 
-### Awesome Feature 2
+   ### Awesome Feature 2
 
-Description here.
+   Description here.
 
-## Changes
+   ## Changes
 
-* #123 Make it better - thanks @SlackHQ
-* #123 Fix something wrong - thanks @seratch
-```
+   * #123 Make it better - thanks @SlackHQ
+   * #123 Fix something wrong - thanks @seratch
+   ```
 
 3. (Slack Internal) Communicate the release internally
-
-- Include a link to the GitHub release
+   - Include a link to the GitHub release
 
 4. Make announcements
-
-- #slack-api in dev4slack.slack.com
-- #tools-bolt in community.slack.com
+   - #slack-api in dev4slack.slack.com
```

Comment:
It looks like this workspace has been deleted.

Reply:
Yes, @eddyg is right. @WilliamBergamin can you remove this step?

Reply:
Good catch thanks @eddyg 💯